arangod/Replication2/ReplicatedLog/NetworkMessages.cpp

```diff
@@ -298,11 +298,10 @@ auto replicated_log::AppendEntriesRequest::fromVelocyPack(
   auto entries = std::invoke([&] {
     auto entriesVp = velocypack::ArrayIterator(slice.get("entries"));
     auto transientEntries = EntryContainer::transient_type{};
-    std::transform(
-        entriesVp.begin(), entriesVp.end(),
-        std::back_inserter(transientEntries), [](auto const& it) {
-          return InMemoryLogEntry(PersistingLogEntry::fromVelocyPack(it));
-        });
+    for (auto it : entriesVp) {
```

Comment:
transientEntries.reserve

Reply:
`reserve` function is not available here. This is not an std container, but an immer container.